docs/modules/ROOT/pages/subscriptions/events/connect.adoc

```diff
@@ -0,0 +1,698 @@
+                name: "Keanu Reeves"
+            }
+        },
+        directors: null
+    }
+},
+{
+    # 2 - actorRelationshipCreated
+    event: "DISCONNECT"
+    timestamp
+    actor {
+        name: "Keanu Reeves"
+    }
+    createdRelationship {
+        movies: {
+            screenTime: 420,
+            node: {
+                title: "John Wick",
+                genre: "Action"
+            }
+        }
+    }
+}
+```
+
+==== Person
+
+As the `Person` type does not define any relationships, it is **not** possible to subscribe to `CONNECT` events for this type.
+
+NOTE: As we know from Conclusion#2, the relationship of type `DIRECTED` is **not** reciprocal. 
+
+Therefore, when relationships of this type are made, such as by running a mutation as follows...
+```graphql
+mutation {
+    createMovies(
+        input: [
+            {
+                directors: {
+                    Actor: { # relationship 1
+                        create: [
+                            {
+                                node: {
+                                    name: "Woody Allen"
+                                },
+                                edge: {
+                                    year: 1989
+                                }
+                            }
+                        ]
+                    },
+                    Person: { # relationship 2
+                        create: [
+                            {
+                                node: {
+                                    name: "Francis Ford Coppola",
+                                    reputation: 100
+                                },
+                                edge: {
+                                    year: 1989
+                                }
+                            }
+                        ]
+                    }   
+                },
+                title: "New York Stories",
+                genre: "Comedy"
+            }
+        ]
+    ) {
+        movies {
+            title
+            genre
+        }
+    }
+}
+```
+
+...two events will potentially be published, depending on whether we subscribed to `CONNECT` events on the `Movie` type:
+```graphql
+{ 
+    # relationship 1
+    event: "DISCONNECT"
+    timestamp
+    movie {
+        title: "New York Stories",
+        genre: "Comedy"
+    }
+    createdRelationship {
+        actors: null,
+        directors: {
+            year: 1989,
+            node: {
+                name: "Woody Allen"
+            }
+        }
+    }
+},
+{ 
+    # relationship 2
+    event: "DISCONNECT"
+    timestamp
+    movie {
+        title: "New York Stories",
+        genre: "Comedy"
+    }
+    createdRelationship {
+        actors: null,
+        directors: {
+            year: 1989,
+            node: {
+                 name: "Francis Ford Coppola",
+                reputation: 100
+            }
+        }
+    }
+}
+```
+
+[[connect-same-label]]
+=== Types using the same Neo4j label
```

Comment:
We should move this into a `Considerations` (or similar) section, as this is not really the recommended approach on designing an API and more of an edge case